env_test.go

```diff
@@ -571,6 +571,82 @@ func TestTextUnmarshalerError(t *testing.T) {
 	assert.Error(t, env.Parse(cfg))
 }
 
+func TestMinMaxBorders(t *testing.T) {
```

Comment:
could you please add a test to mix it with other ones ? like envExpand ?

Reply:
done in 01fb3a8309094cd594c7f7d2ec8483f13a6059eb